airflow/models/variable.py

```diff
@@ -47,12 +47,14 @@ class Variable(Base, LoggingMixin):
     id = Column(Integer, primary_key=True)
     key = Column(String(ID_LEN), unique=True)
     _val = Column('val', Text)
+    description = Column(Text(5000))
```

Comment:
```suggestion
    description = Column(Text)
```

@itroulli sorry for the confusion. Postgres doesn't allow modifier on `text` field. Can you also update this and the alembic migration file? (you would need to `git pull` the branch first) Let me know if you have any questions :)

Reply:
I think I'd need some help with updating the revision keys in the migration file. I believe this is why some checks are failing. With Python 3.6, MySQL 5.7 right now I get this error:
`alembic.util.exc.CommandError: Can't locate revision identified by '458c2838cbca'`

Reply:
Hmm, this is because there are other migration changes merged to the master before you, which happens sometimes given the number of PRs we have. We used to have problems with multiple diverge alembic heads, and we mitigate it by making sure that any PR affecting the `migration` folder will need to rebase on the latest master. So, there are two ways of doing this:

- manually change the `down_revision` number to the latest revision number in `master` branch
- delete the revision file and regenerate it after rebase to `master` (I personally do this.)

Reply:
This is the most recent change https://github.com/apache/airflow/pull/15203

Reply:
Seems that everything is passing now! :)